meson.build

```diff
@@ -1252,8 +1256,9 @@ default_dnssec = get_option('default-dnssec')
 if skip_deps
         default_dnssec = 'no'
 endif
-if default_dnssec != 'no' and conf.get('HAVE_GCRYPT') == 0
-        message('default-dnssec cannot be set to yes or allow-downgrade when gcrypt is disabled. Setting default-dnssec to no.')
+if default_dnssec != 'no' and conf.get('HAVE_OPENSSL') == 0
+        message('default-dnssec cannot be set to yes or allow-downgrade when
```

Comment:
```
WARNING: Newline character in a string detected, use ''' (three single quotes) for multiline strings instead.
This will become a hard error in a future Meson release.         message('default-dnssec cannot be set to yes or allow-downgrade when 1262 16
```